### PR TITLE
45 changed Occupation label to Occupation/Role

### PR DESCRIPTION
**Pull request by kamerynB**

### CODE REVIEW

File: client/src/lib/parse/data/PersonAndGroupParser.ts [hunk 1]
```diff
@@ -259,7 +259,7 @@ export default class PersonAndGroupParser extends EntityParser {
             if (eq.id === config.aat.gender) {
               label = 'Gender'
             } else if (eq.id === config.aat.occupation) {
```
Comment (clarkepeterf) [inline client/src/lib/parse/data/PersonAndGroupParser.ts:261]: @kamerynB there is another AAT for Role, which the frontend doesn't have right now. (It is http://vocab.getty.edu/aat/300435108). The ML update added this AAT. I think maybe the frontend do the same? Like check if `eq.id === config.aat.occupation || eq.id === config.aat.role`?

**Comment by clarkepeterf** [comment]
There is also a label in /src/config/personLabels.ts that is 'Occupations' - it appears that label isn't used anywhere right now. But perhaps we should change the label and then use that where the Occupation/Role label is used so that it is a centralized change?

**Comment by clarkepeterf** [comment]
There is also an Occupations label in src/features/results/PersonSnippet.tsx

We should potentially change that label as well. But that would correspond with changing the PersonAndGroupParser `getOccupations()` function.

Consider changing these as well

File: client/src/lib/parse/data/PersonAndGroupParser.ts (hunk 1 above)
Comment (kamerynB) [reply client/src/lib/parse/data/PersonAndGroupParser.ts]: This should be fixed now.

**Pull request merged**